packages/core/src/application.ts

```diff
@@ -234,16 +237,28 @@ export class Application extends Context {
   ) {
     options = options || {version: '2.0', format: 'json'};
     let specObj = this.getApiSpec();
+    let appPort = await this.get('http.port');
+    // TODO make this more robust i.e. get host, protocol, etc. dynamically
+    let appUrl = 'http://127.0.0.1:' + appPort + '/swagger';
     if (options.version === '3.0.0') {
       specObj = await swagger2openapi.convertObj(specObj, {direct: true});
+      appUrl.replace('swagger', 'openapi');
     }
     if (options.format === 'json') {
       const spec = JSON.stringify(specObj, null, 2);
+      appUrl += '.json';
       response.setHeader('content-type', 'application/json; charset=utf-8');
+      response.statusCode = 308;
+      response.setHeader('Location', 'http://petstore.swagger.io/?url='
+      + appUrl);
```

Comment:
This is wrong. We should not deal with redirect here for the swagger/openapi endpoint. Our responsibility is to serve specs.

Reply:
Maybe I misunderstood the story, but I thought we should do it as part of the following acceptance criteria:
```
Pass generated swagger.json to editor.swagger.io so that users can leverage the existing editor tools on their local application.
```

Reply:
That was my understanding, too. @raymondfeng How do we pass the information along to the server at `editor.swagger.io` without modifying the request?

Reply:
There are two separate concerns:
1. Have endpoints from LB to serve openapi/swagger specs
2. A UI to render the spec as api explorer

Most UI such as `editor.swagger.io` has an option to import URL or set the location. If we want to use redirect to `editor.swagger.io`, the path should not be `/swagger.json`. At lease we need to have something like `/swagger.json?ui=true`. It should be replacing 1 at all.

For the GA version, we'll have `/explorer` to serve the UI.

Reply:
Okay it makes sense to me now that we need to preserve the `/{openapi,swagger}.{json,yaml}` endpoints. I will take a crack at implementing `/swagger.json?ui=true`.